src/db/repository/postgres/CommunityPostsRepository.ts

```diff
@@ -0,0 +1,34 @@
+import { Kysely } from 'kysely';
+
+import { DB, CommunityPosts } from '../../types/generated/database';
+
+export type CommunityPostsCreateParams = Omit<CommunityPosts, 'id' | 'created_at'>;
+
+export default class CommunityPostsRepository {
+	constructor(private readonly database: Kysely<DB>) {}
+
+	getAll() {
+		return this.database
+			.selectFrom('community_posts')
+			.innerJoin('agencies', 'community_posts.id', 'agencies.id')
+			.selectAll('community_posts')
+			.execute();
+	}
+
+	getAllByVerifiedAgencies() {
+		return this.database
+			.selectFrom('community_posts')
+			.innerJoin('agencies', 'community_posts.id', 'agencies.id')
+			.where('agencies.is_verified', '=', true)
+			.selectAll('community_posts')
+			.execute();
+	}
```

Comment:
```suggestion
	getByAgencyVerificationStatus(is_verified: boolean) {
		return this.database
			.selectFrom('community_posts')
			.innerJoin('agencies', 'community_posts.id', 'agencies.id')
			.where('agencies.is_verified', '=', is_verified)
			.selectAll('community_posts')
			.execute();
	}
```

We should make this accept a status value as a param so we don't need to create another function later on for the opposite version.